DiagnosticsExtension/Models/ConnectionStringValidator/ConnectionStringType.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+
+namespace DiagnosticsExtension.Models.ConnectionStringValidator
+{
+    public enum ConnectionStringType
```

Comment:
Please add (or I can add in my PR):
ServiceBus
EventHubs
CosmosDB

Reply:
done